options.js

```diff
@@ -88,88 +94,123 @@ function updateShortcutInputText(inputId, keyCode) {
   document.getElementById(inputId).keyCode = keyCode;
 }
 
+function updateCustomShortcutInputText(inputItem, keyCode) {
+  inputItem.value = keyCodeAliases[keyCode] || String.fromCharCode(keyCode);
+  inputItem.keyCode = keyCode;
+}
+
+function add_shortcut() {
+  var html = '<select class="customDo">' +
+    '<option value="slower">Decrease speed</option>' +
+    '<option value="faster">Increase speed</option>' +
+    '<option value="rewind">Rewind</option>' +
+    '<option value="advance">Advance</option>' +
+    // '<option value="reset">Reset speed</option>' +
+    '<option value="fast">Preferred speed</option>' +
+    '<option value="muted">Muted</option>' +
+    '<option value="fast">Pause</option>' +
+    '</select> ' +
+    '<input class="customKey" type="text" placeholder="press a key"/> ' +
+    '<input class="customValue" type="text" placeholder="value (0.10)"/> ' +
+    '<select class="customReturn">' +
+    '<option value="0">Do not disable website key bindings</option>' +
+    '<option value="1">Disable websites key bindings</option>' +
+    '</select>' +
+    '<button class="removeParent">X</button>';
+  var div = document.createElement('div');
+  div.setAttribute('class', 'row customs');
+  div.innerHTML = html;
+  var customs_element = document.getElementById("customs");
+  customs_element.insertBefore(div, customs_element.children[customs_element.childElementCount - 1]);
+}
+
+function createKeyBindings() {
+  var doo = $(this).find(".customDo").val();
```

Comment:
We can use querySelector here, no need for jQuery.

See: https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector